src/components/SidePanel/SidePanel.layouts.jsx

```diff
@@ -19,32 +19,12 @@ export function HeaderAndFooter({
   const headerRef = useRef(null)
   const bodyRef = useRef(null)
   const footerRef = useRef(null)
-
-  useEffect(() => {
-    setTimeout(() => {
-      const { isBottomScrolled } = handleShadows(bodyRef)
-
-      if (isBottomScrolled) {
-        footerRef.current.classList.add('with-shadow')
-      }
-    }, 100)
-  }, [bodyRef])
-
-  const handleOnScroll = throttle(e => {
-    const { isTopScrolled, isBottomScrolled } = handleShadows(bodyRef)
-
-    if (isTopScrolled) {
-      headerRef.current.classList.add('with-shadow')
-    } else {
-      headerRef.current.classList.remove('with-shadow')
-    }
-
-    if (isBottomScrolled) {
-      footerRef.current.classList.add('with-shadow')
-    } else {
-      footerRef.current.classList.remove('with-shadow')
-    }
-  }, 100)
+  const [handleOnScroll] = useScrollShadow({
+    bottomRef: footerRef,
+    scrollableRef: bodyRef,
+    topRef: headerRef,
+    shadows: { initial: 'none' },
```

Comment:
There has been one issue (even before, but still present), that when there is scroll, the shadow at the bottom does not appear until the content is scrolled. This can be misleading users that there is nothing to scroll to (especially when scrollbar is not visible). Here's example: https://c.hlp.sc/xQujRO7Q

Do you think this setting needs to be changed or maybe it's some different issue to be solved?